projects/scion/toolkit/viewport/src/viewport.component.spec.ts

```diff
@@ -905,11 +908,11 @@ describe('Viewport', () => {
       'height': '9000px',
       'width': '9000px',
     });
-    fixture.detectChanges();
+    await flushChanges(fixture);
 
     expect(getSize(fixture, 'sci-viewport')).toEqual(jasmine.objectContaining({height: 300, width: 500}));
-    expect(isScrollbarVisible(fixture, 'vertical')).toBeTrue();
-    expect(isScrollbarVisible(fixture, 'horizontal')).toBeTrue();
+    expect(isScrollbarVisible(fixture, 'vertical')).toBe(true);
```

Comment:
Use `toBeTrue` on both expectations since you don't provide an expectation fail output.

Reply:
done